### DISCUSSION TITLE
authentication

### DISCUSSION THREAD

**Discussion by Sunilkumar-Halasur** [community/community#45245] [New to GitHub] [answered]
when i'am cloning my repo, getting error like this 
" remote: Support for password authentication was removed on August 13, 2021."

**Comment by jge162** [comment, marked as answer]
@sunilakumara-smartenovations Starting from August 13, 2021, GitHub removed support for using a password to authenticate Git operations. This means that you will need to use an SSH key or a Personal Access Token (PAT) to authenticate when cloning a repository.

If you haven't already, you'll need to generate an SSH key and add it to your GitHub account. You can follow the instructions on this link: https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

**Reply by Sunilkumar-Halasur** [reply]
Thank you @jge162

**Reply by Annytech001** [reply]
i regenerated my token recently and this happened what should i do?

**Reply by Sunilkumar-Halasur** [reply]
> i regenerated my token recently and this happened what should i do?
> 

follow below procedure
settings > developer settings > personal access token >  token(classic) > generate new token > generate new token(classic) > expiration = no expiration and select repo then generate token > here copy your "Token"


open the terminal enter below command

>git remote set-url origin https://"Token"@github.com/ "repository url"

then clone it using
>git clone  https://"Token"@github.com/ "repository url" 
 

May be it will help you

**Reply by goever1** [reply]
Hi, I tried what you said. how ever it does not work,
this is the messaged that shows:
git: 'remoted' is not a git command. See 'git --help'.                                                                                                                                        
                                                                                                                                                                                              
The most similar command is                                                                                                                                                                   
        remote-fd

**Reply by Sunilkumar-Halasur** [reply]
> Hi, I tried what you said. how ever it does not work,
> this is the messaged that shows:
> git: 'remoted' is not a git command. See 'git --help'.                                                                                                                                        
>                                                                                                                                                                                               
> The most similar command is                                                                                                                                                                   
>         remote-fd                    

@goever1  May be you enter wrong command, instead of "remoted" use "remote"

**Reply by keep57** [reply]
Thanks!!!

**Reply by imanthangadhiya** [reply]
Below is the correct command after the personal access token is generated

git remote add origin https://KEY@github.com/RepoURL
then
git push -u origin main

**Comment by ghost** [comment]
If you no longer have access to the email address associated with your GitHub account.
Please help me to access my account again I have no reference to that GitHub account so I need my account because a lot of effort I do with my GitHub account please help me now I create a new GitHub account but I need that email account 

my GitHub account URL is

```https://github.com/Kashif1Naqvi```
associated with Gmail id [mrkashif029@gmail.com](mailto:mrkashif029@gmail.com)
but I forget my email password so I don't change my GitHub password without my Gmail password.
please help me to recover my account

**Reply by barkatansari2022** [reply]
hey, i am also facing similar problems with my account.
https://github.com/baraktullah 
my Email id: b.ansari2010@gmail.com 
now i am unable to login , unable to reset password 
Please help to reset my password either how i can use this Email address in new account??

**Comment by barkatansari2022** [comment]
Hey guys,
 i am also facing problems with my GitHub account.  url:  https://github.com/baraktullah (it was created last 2013 & was not in use)
my Email id:  b.ansari2010@gmail.com
now i am unable to login , unable to reset password
Please help to reset my password either how i can use this Email address in new account??

**Comment by JJESSUUSS** [comment]
Enviado desde Yahoo Mail para iPhone


El miércoles, febrero 15, 2023, 6:07 a.m., barkatansari2022 ***@***.***> escribió:




hey, i am also facing similar problems with my account.
https://github.com/baraktullah
my Email id: ***@***.***
now i am unable to login , unable to reset password
Please help to reset my password either how i can use this Email address in new account??

**Comment by mdxabu** [comment]
is there any permanent solution for this without using PAT
